opacplot2/utils.py

```
@@ -119,7 +119,7 @@ def interpDT(arr, dens, temps,
     # "When on a regular grid with x.size = m and y.size = n,
     # if z.ndim ==2, then z must have shape (n, m)."
     # arr will have shape (dens.size, temps.size), so we must transpose it.
-    f = sp.interpolate.interp2d(dens, temps, arr.T)
+    f = sp.interpolate.interp2d(dens, temps, arr.T, kind=kind)
```

This should raise an exception if `kind!='linear'` as that would not be thermodynamically consistent.